security.html

```diff
@@ -0,0 +1,127 @@
+                    template: "lib/infusion/framework/preferences/html/SeparatedPanelPrefsEditor.html"
+                });
+            });
+        </script>
+
+        <!-- BEGIN markup for Preference Editor -->
+        <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!-- This div is for the sliding panel that shows and hides the Preference Editor controls -->
+            <div class="fl-panelBar">
+                <span class="fl-prefsEditor-buttons">
+                    <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                    <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+            <a href="#navigation">Skip to Navigation</a>
+        </div>
+        
+        <div class="floe">
+            <div class="floe-header">
+                <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
+                    <span class="floe-header-tagline">
+                        flexible learning for open education
+                    </span></a>
+                </h1>
+            </div>
+
+            <div class="floe-content floe-video">
+                <div class="flc-toc-tocContainer toc"> </div>
+                <div class="floe-breadcrumbs"><a href="index.html">&lt; back</a></div>
+                <h2> Security and Privacy Architecture </h2>
+    
+                <p>The Floe Project is working with the
+                    <a href="http://gpii.net">Global Public Inclusive Infrastructure</a>
+                    to provide learners with friendly and comprehensive tools for sharing their data
+                    with the web sites, access features and assistive technologies that they trust.
+                    These tools will give learners complete control over which preferences are shared
+                    with any GPII-enabled services.</p>
+                <p>For more information and to see an example of the security architecture in action,
+                    please see the video below.</p>
+                <video controls width="640px" height="360px" tabindex="0">
+                        <source src="https://idrc.cachefly.net/floeproject.org/videos/Floe-GPII-security.mp4" type="video/mp4"></source>
+                        <source src="https://idrc.cachefly.net/floeproject.org/videos/Floe-GPII-security.webm" type="video/webm"></source>
+                </video>
+            </div>
+        
+            <div class="floe-footer">
```

Comment:
can we make this a `<footer>`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer